integration-cli/requirements.go

```diff
@@ -88,3 +96,14 @@ func testRequires(t *testing.T, requirements ...TestRequirement) {
 		}
 	}
 }
+
+// testRequires checks if the environment satisfies the requirements
+// for the test to run or skips the tests.
+func testRequiresWithLog(t *testing.T, msg string, requirements ...TestRequirement) {
```

Comment:
Few things are not clear yet to me. I am ready to change this code or refactor and merge this into the existing method call but would like to see few more opinions.
What happens when I simply do "make test" ? Do I see clean output saying SKIP: ...?
Secondly, I advocate we have [PASSED], [SKIPPED] or [NOTIMPLEMENTED or something if go permits] for all tests and that is easy to track using tools?